package.json

```diff
@@ -11,6 +11,8 @@
     }
   },
   "dependencies": {
+    "@noble/curves": "^1.8.1",
+    "@scure/base": "^1.2.4",
```

Comment:
btw do we need this dependencies only for the base64? I think crypto can handle that https://nodejs.org/api/crypto.html#crypto_cipher_final_output_encoding

Reply:
yeah, would be good if they are not needed, i've just copied the approaches from ccxt.
so we would be able to update ccxt too.

Reply:
@ttodua not needed, you can check now